### ISSUE TITLE
Add doc how to show hidden (dot) files

### ISSUE THREAD

**Issue by Natherul** [AstroNvim/astronvim.github.io#69]
### Is your feature related to a problem?

Yes, I cant find any documentation on how its done. Its been asked several times in git (in now closed issues) but even following those I cannot seem to be able to do this.

As example:
https://github.com/AstroNvim/AstroNvim/issues/631
Contains code snippets that does nothing, or Im simply not understanding them. 

### Describe the new feature

Write clear documentation somewhere on https://astronvim.com/ with what file to edit and how to edit it. (with full file example would be best)

**Comment by mehalter** [comment]
This can be found with `?` in the neo-tree filetree or you can look at the `neo-tree` documentation. We are trying to minimize the duplicated documentation when the plugins themselves will have infinitely better documentation than we will write.

TL;DR `H` will toggle hidden files

**Comment by Natherul** [comment]
The TLDR does not apply. Im looking for a PERMANENT setting. I will however look at the documentation mentioned later.
